### DISCUSSION TITLE
Verify Domains & Click Tracking (Admin Center) - (Office hours: Thurs, Feb 11)

### DISCUSSION THREAD

**Discussion by JT1989** [twilio-labs/paste#1156] [Office hours] [answered]
**Background:** We want customers to verify their domain and setup their domain to shorten URLs within messages and track engagement data
**Research results:** https://docs.google.com/presentation/d/1ZWvFy6u2DN_MbmLXeo8xKfqwkjG_7DC2xvQpppiTXw4/edit
**Prototype:** https://twilio.invisionapp.com/share/2ZZZWB6WD5J
**Stage of design**: Hi-Fi Prototyping
**Request:** I invite you to weigh in on the designs, please provide specific feedback on correct usage of Paste components and accessibility concerns.
**Notes:** I have met with Sean Mateer and Priyanka, Priyanka is fully aware of these designs.

**Comment by kate-porter** [comment, marked as answer]
Hey @JT1989 - thanks for sharing this work, we'll be sure to take a look before office hours so we've got good context going into that meeting. I'm taking a look through it now and might drop in some comments while I'm in there. :)

In the meantime, if you have any specific parts of the flow where you _definitely_ want to get our feedback (e.g. areas where you are uncertain about component usage or you are not sure if something is designed in an accessible way), that would be helpful to know, so we can be sure to cover those.
